src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/Validator.cs

```diff
@@ -236,7 +237,7 @@ public static void ValidateProperty(object value, ValidationContext validationCo
             var err = GetValidationErrors(value, validationContext, attributes, false).FirstOrDefault();
             if (err != null)
             {
-                err.ThrowValidationException();
+                throw err.GetValidationException();
```

Comment:
Why did you change these?  If it's purely about getting to 100% code coverage by allowing the compiler to see that the subsequent brace is unreachable, that's not worth the change.